FordLabs/PeopleMover
# redirect to 403 & 404page

## Issue
Resolves 48 & 11

## What was done
- [x] redirect to 404 page when space does not exist
- [x] redirect to 403 page when user does not have access to space

## How to test
1. Try accessing a space that do not exist and you should get redirected to 404 page
2. Try accessing a space that you do not have access and read only is off, you should see the 403 page
